retriever/engines/sqlite.py

```diff
@@ -52,6 +52,42 @@ def get_bulk_insert_statement(self):
         insert_stmt = insert_stmt.rstrip(", ") + ")"
         return insert_stmt
 
+    def insert_data_from_file(self, filename):
+        """Perform a high speed bulk insert
+
+        Checks to see if a given file can be bulk inserted, and if so loads
+        it in chunks and inserts those chunks into the database using
+        executemany.
+        """
+        CHUNK_SIZE = 1000000
+        self.get_cursor()
+        ct = len([True for c in self.table.columns if c[1][0][:3] == "ct-"]) != 0
```

Comment:
A comment to describe this operation would be useful. Also, a descriptive variable name would be better.